proton

```diff
@@ -148,10 +148,15 @@ def try_get_game_library_dir():
 #
 # On failure, returns None
 def find_nvidia_wine_dll_dir():
-    libdl = CDLL(find_library("libdl"))
-    libglx_nvidia = CDLL("libGLX_nvidia.so.0")
+    try:
+        libdl_soname = find_library("dl")
```

Comment:
I think you should check for `None` here as well, at least that's what the documentations will be returned if no library can be found.

Reply:
Ah yeah, I see that here on a second pass-through: https://docs.python.org/3/library/ctypes.html#finding-shared-libraries